packages/web/src/scss/components/UNSTABLE_EmptyState/README.md

```diff
@@ -22,7 +24,7 @@ The `UNSTABLE_EmptyState` component is a main container responsible for position
 
 This component is based on the [Stack][stack] component and accepts all its variants. For more information about `Stack` usage options, see the [Stack documentation][stack].
 
-## UNSTABLE EmptyStateSection
+### UNSTABLE_EmptyStateSection
```

Comment:
this should be the subtitle of the `UNSTABLE_EmptyState`, shouldn't it?

Reply:
Well, I am not quite sure. I think this is ok.

```
Component Composition
  UNSTABLE_EmptyState
  UNSTABLE_EmptyStateSection
```

I think this will show that there are two components used to compose the entire EmptyState.

Reply:
ok